kpops/components/base_components/kafka_app.py

```diff
@@ -117,6 +117,16 @@ async def clean(self, dry_run: bool) -> None:
             log.info(f"Uninstall cleanup job for {self.helm_release_name}")
             await self.destroy(dry_run)
 
+    async def update_cleaner_with_cluster_values(self) -> None:
+        """Update cleaner with cluster values if the release exists."""
+        cluster_values = await self.helm.get_values(
+            self.namespace, self.helm_release_name
+        )
+        if cluster_values:
+            self.app = self.app.__class__.model_validate(cluster_values)
+            self.app.name_override = self.helm_name_override
```

Comment:
perhaps we could consider setting it to `None` here? Then the behavior is aligned with our other components, where it gets assigned during `to_helm_values` https://github.com/bakdata/kpops/blob/e692e8245524300a163bd769fe0b10fb58918f49/kpops/components/base_components/helm_app.py#L189

Reply:
discussed it further and agreed against it